### ISSUE TITLE
scan reports can provide more information?

### ISSUE THREAD

**Issue by PeterLee4532** [zaproxy/zaproxy#1976]
Including scanning start / end time / spend time scanning the site, scanning project, scanning project description
Scanning project success or failure .... and so on.

**Comment by psiinon** [comment]
And what rules were used when scanning + at what levels.

**Comment by kingthorin** [comment]
What if the "session" represents or includes, multiple scans?

What if the session is saved and re-opened in the future? Are all those details persisted?

Not that any of this should prevent Fulfillment of this request, just that they should be considered...

**Comment by psiinon** [comment]
Yeah. We _really_ need to think about how we'd like reporting to work - theres lots to do and a grand plan would probably really help :/

**Comment by afatac** [comment]
I think making the report showing all scanned vulnerabilities (not just those being alerted) will be very important for using it as a proof when going through a security audition.

**Comment by thc202** [comment]
Added to #6483.
